python/ray/_private/event/export_event_logger.py

```diff
@@ -12,24 +13,47 @@
 from ray.core.generated.export_submission_job_event_pb2 import (
     ExportSubmissionJobEventData,
 )
+from ray.core.generated.export_train_state_pb2 import (
+    ExportTrainRunEventData,
+    ExportTrainRunAttemptEventData,
+)
 from ray._private.protobuf_compat import message_to_dict
 
 global_logger = logging.getLogger(__name__)
 
 # This contains the union of export event data types which emit events
 # using the python ExportEventLoggerAdapter
-ExportEventDataType = Union[ExportSubmissionJobEventData]
+ExportEventDataType = Union[
+    ExportSubmissionJobEventData,
+    ExportTrainRunEventData,
+    ExportTrainRunAttemptEventData,
+]
+
+
+class EventLogType(Enum):
+    TRAIN_STATE = (
+        "train_state",
+        {ExportTrainRunEventData, ExportTrainRunAttemptEventData},
+    )
+    SUBMISSION_JOB = ("submission_job", {ExportSubmissionJobEventData})
```

Comment:
@nikitavemuri can you double check the changes introduced here? I wasn't sure if I needed to maintain the "name" used for submission job - I think this was `EVENT_SUBMISSION_JOB`?

Reply:
I'm not sure if anyone is using it, but would probably be safer to keep the same name. The submission job events currently go to `event_EXPORT_SUBMISSION_JOB.log`, so `log_type_name` here should be `EXPORT_SUBMISSION_JOB`. The rest of this interface looks good!